spec/ttml3.xml

```diff
@@ -2500,7 +2540,11 @@ namespace names are listed in <specref ref="namespace-vocab-table"/>.
 <head>TTML Content Document Type</head>
 <p>The TTML Content Document Type is an <loc href="#terms-abstract-document-type">abstract document type</loc> of a profile
 of the Timed Text Markup Language intended to be used for interchange among distribution systems. This document type
-is defined in terms of the element and attribute vocabulary specified in <specref ref="vocabulary"/>.</p>
+is defined in terms of <loc href="#terms-module">modules</loc> that specify element and attribute vocabulary,
```

Comment:
Can we be more succint? The definition of module already says all of this. Please delete:
"that specify element and attribute vocabulary,
features, and other semantic behaviors, where such modules take the form of either (or both)
<loc href="#terms-internal-module">internal modules</loc> defined (internally) within this document
or <loc href="#terms-external-module">external modules</loc> defined (externally) with respect to this document,
and where each module is listed in the <loc href="#terms-module-registry">module registry</loc>"

Reply:
Deleted in https://github.com/w3c/ttml3/pull/30/commits/eeb4d4e77e42a291ff17c711651d396cde457cff.